<launch>
<node pkg="l3xz_mapping" type="knownposition_node" name="lidar_mapper" output="screen" ns="l3xz">
	<param name="pixel_x" value="600"/>
	<param name="pixel_y" value="600"/>
	<param name="resolution" value="0.1"/>
    <param name="lidar_topic" value="/lidar"/>
    <param name="grid_topic" value="/grid"/>
    <param name="odometry_topic" value="/odom"/>
    <param name="rate_hz" value="5"/>
    <param name="show" value="false"/>
</node>
</launch>
